flytekit/annotated/python_function_task.py

```diff
@@ -122,6 +122,38 @@ def get_container(self, settings: RegistrationSettings) -> _task_model.Container
         )
 
 
+def isnested(func) -> bool:
```

Comment:
extreme nit but can we use snake case for these method names for readability?

Reply:
i just followed the style in inspect. I was going to use snake case, what do you think
they use. issubclass, isclass, isobject etc

Reply:
TIL. ignore me then

Reply:
resolving